### DISCUSSION TITLE
Clinical notes

### DISCUSSION THREAD

**Discussion by arvindshmicrosoft** [synthetichealth/synthea#1415] [General]
Is there a 1:1 association between clinical notes and medical encounters? When `exporter.clinical_note.export` is set to `true`, I can see that in the `notes` sub-folder, there is one text file produced per patient. I take it that this is the cumulative set of notes for this patient. It looks like dates are used as the "separator" between different notes. Parsing this is not very convenient, so I was wondering if there is any way to export just the notes as JSON files perhaps? Or even one (text) file per encounter (assuming that's the degree of association)?

**Comment by jawalonoski** [comment]
Yes, the notes are 1:1 with encounters.

No, there is no other setting that separates them into different files or JSON entries.

The notes are base64 encoded in FHIR output as well, so if you have the FHIR, look for the DocumentReference resource associated with each Encounter.
